### PR TITLE
feat(1113): Parse meta from config on event creation

### PR DESCRIPTION
**Pull request by Filbird**
## Context
Events should be created with the meta specified in the config.

## Objective
Create an `event` with `meta` if the field is populated.

## Reference
https://github.com/screwdriver-cd/screwdriver/issues/1113

### CODE REVIEW

**Review by minzcmu — APPROVED** [review]
tests?

**Comment by Filbird** [comment]
:^)

**Review by d2lam — APPROVED** [review]

**Comment by minzcmu** [comment]
Can you also add meta as an optional field to the jsDoc for buildFactory.create function? https://github.com/screwdriver-cd/models/blob/master/lib/buildFactory.js#L155
And probably set the default value `{}` there too. Just to be consistent

**Comment by Filbird** [comment]
Ooh. Yep! Good catch

**Review by minzcmu — APPROVED** [review]

File: test/lib/buildFactory.test.js [hunk 1]
```diff
@@ -133,7 +133,7 @@ describe('Build Factory', () => {
         });
     });
 
-    describe('create', () => {
```
Comment (jithine) [inline test/lib/buildFactory.test.js]: remove `only`

**Review by jithine — CHANGES_REQUESTED** [review]

**Review by jithine — APPROVED** [review]

**Pull request merged**